packages/manager/src/features/Kubernetes/KubernetesClusterDetail/UpgradeClusterDialog.tsx

```diff
@@ -9,6 +9,17 @@ import useKubernetesClusters from 'src/hooks/useKubernetesClusters';
 import { HIGH_AVAILABILITY_PRICE } from 'src/constants';
 import { HACopy } from '../KubeCheckoutBar/HACheckbox';
 import { useSnackbar } from 'notistack';
+import { makeStyles, Theme } from 'src/components/core/styles';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  noticeHeader: {
+    fontWeight: 'bold',
+  },
+  noticeList: {
+    paddingLeft: 16,
+    marginTop: 8,
```

Comment:
If we're using multiples of 8, we can use `theme.spacing()` instead

```suggestion
    paddingLeft: theme.spacing(2),
    marginTop: theme.spacing(),
```